### DISCUSSION TITLE
Cask OpenConnect-GUI should be un-deprecated

### DISCUSSION THREAD

**Discussion by elfurbe** [Homebrew/discussions#5107] [Casks]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

The OpenConnect-GUI cask was deprecated (https://github.com/Homebrew/homebrew-cask/blob/81613c612fc191aada0941152c627795edcdc5ce/Casks/o/openconnect-gui.rb) due to upstream development being discontinued, which was true at the time. However they have put out a new release (https://gitlab.com/openconnect/openconnect-gui/-/releases) and it appears to be under relatively active development for the last couple months (https://gitlab.com/openconnect/openconnect-gui/-/commits/main/?ref_type=HEADS), so I think it should be re-instated. Not sure if there's a proper way to report that change to a previously deprecated formula, but I've naively un-deleted enough things on the internet to know someone will likely be disproportionately incensed if I try to undo their great work with a mere pull request, so I figured I'd make a discussion post and find out whose honor I have to challenge to not have to manage my installation of this tool by hand like a savage.

### Relevant casks

openconnect-gui

**Comment by gromgit** [comment]
There's only been one new release (1.6.0), and there's no corresponding macOS binary, which I think would be the bare minimum for a _cask_ to be undeprecated.
